### PR TITLE
Update db.count()

### LINKED ISSUE #710: Use app.db.count() instead of app.count()
### 🐛 Describe the bug

Following code snippet gives a warning

```
from embedchain import App
app = App()
app.count()
[root] [WARNING] DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.
```

Count function counts the number of chunks in the database. We need to update and fix this warning.

### PR DESCRIPTION
**Pull request by th-shristi**
Updated db.count() function inplace of count()

## Description
Fixes #710 
--> Replaced count() with db.count()
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [ ] Test Script (please provide)

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [x ] closes #710  (Replace xxxx with the GitHub issue number)
- [x ] Made sure Checks passed

### CODE REVIEW

File: embedchain/embedchain.py [hunk 1]
```diff
@@ -521,17 +521,19 @@ def set_collection_name(self, name: str):
...
-        :return: The number of embeddings.
-        :rtype: int
-        """
-        logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
-        return self.db.count()
+        # :return: The number of embeddings.
+        # :rtype: int
+        # """
+        # logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
+
```
Comment (deshraj) [inline embedchain/embedchain.py:536]: Can you please remove the commented lines since we no longer will use it?

**Review by th-shristi — COMMENTED** [review]
Removed the comments

File: embedchain/embedchain.py (hunk 1 above)
Comment (th-shristi) [reply embedchain/embedchain.py:548]: @deshraj sure, done!

File: embedchain/embedchain.py [hunk 2]
```diff
@@ -541,17 +541,11 @@ def set_collection_name(self, name: str):
...
-        """
-        Count the number of embeddings.
-
-        DEPRECATED IN FAVOR OF `db.count()`
-
-        :return: The number of embeddings.
-        :rtype: int
-        """
-        logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
+    def db.count(self) -> int:
```
Comment (deshraj) [inline embedchain/embedchain.py:544]: What is the reason for doing this? I am not sure if this is needed. You may just want to remove this function altogether I think.

**Comment by deshraj** [comment]
Closing this PR due to inactivity. Thanks for your contribution.

**Pull request closed without merging**